app/models/post.rb

```diff
@@ -43,13 +43,16 @@ def to_param
   end
 
   def increment_likes
+    return if current_developer == post.developer
```

Comment:
you can move this to a method,  like:

```
def check_autor
  return if current_developer == post.developer
end
```

and use it as a before_action
```
before_action :check_developer, only: [:increment_likes, :decrement_likes]
```

Reply:
Indeed! I'll keep DRY in mind